pkg/settings/registry.go

```diff
@@ -242,8 +242,9 @@ var retiredSettings = map[InternalKey]struct{}{
 	"server.controller.default_tenant.check_service.enabled": {},
 
 	// removed as of 24.2
-	"storage.value_blocks.enabled": {},
-	"kv.gc.sticky_hint.enabled":    {},
+	"storage.value_blocks.enabled":       {},
+	"kv.gc.sticky_hint.enabled":          {},
+	"kv.rangefeed.range_stuck_threshold": {},
```

Comment:
Correct me if I'm wrong - we shouldn't need to worry about mixed version tests here since none use this cluster setting.